core/src/core/transaction.rs

```diff
@@ -1751,13 +1759,10 @@ impl ::std::hash::Hash for Output {
 /// an Output as binary.
 impl Writeable for Output {
 	fn write<W: Writer>(&self, writer: &mut W) -> Result<(), ser::Error> {
-		self.features.write(writer)?;
-		self.commit.write(writer)?;
-		// The hash of an output doesn't include the range proof, which
-		// is committed to separately
-		if writer.serialization_mode() != ser::SerializationMode::Hash {
```

Comment:
Safe to remove this logic?

Reply:
Yes. Thanks for catching this.
We no longer call `hash()` directly on outputs and need to call it explicitly on the output identifier. Outputs are no longer `Hashable`.